--- libvorbis-1.2.0.orig/configure.in	2007-07-25 18:27:00.000000000 +0200
+++ libvorbis-1.2.0/configure.in	2009-05-03 15:34:55.000000000 +0200
@@ -104,9 +104,9 @@
 	case $host in 
 	*86-*-linux*)
 		DEBUG="-g -Wall -W -D_REENTRANT -D__NO_MATH_INLINES -fsigned-char"
-		CFLAGS="-O20 -ffast-math -mno-ieee-fp -D_REENTRANT -fsigned-char"
+		CFLAGS="$CFLAGS -ffast-math -mno-ieee-fp -D_REENTRANT -fsigned-char"
 #              	PROFILE="-Wall -W -pg -g -O20 -ffast-math -D_REENTRANT -fsigned-char -fno-inline -static"
-		PROFILE="-Wall -W -pg -g -O20 -ffast-math -mno-ieee-fp -D_REENTRANT -fsigned-char -fno-inline"
+		PROFILE="-Wall -W -pg -g $CFLAGS -ffast-math -mno-ieee-fp -D_REENTRANT -fsigned-char -fno-inline"
 
 		# glibc < 2.1.3 has a serious FP bug in the math inline header
 		# that will cripple Vorbis.  Look to see if the magic FP stack
@@ -148,13 +148,13 @@
   		PROFILE=${PROFILE}" -D__NO_MATH_INLINES"
   		fi;;
 	powerpc-*-linux*)
-		DEBUG="-g -Wall -W -D_REENTRANT -D__NO_MATH_INLINES"
-		CFLAGS="-O3 -ffast-math -mfused-madd -mcpu=750 -D_REENTRANT"
-		PROFILE="-pg -g -O3 -ffast-math -mfused-madd -mcpu=750 -D_REENTRANT";;
+		DEBUG="-g -Wall -W -D_REENTRANT -D__NO_MATH_INLINES -fsigned-char"
+		CFLAGS="$CFLAGS -ffast-math -mfused-madd -D_REENTRANT -fsigned-char"
+		PROFILE="-pg -g $CFLAGS -ffast-math -mfused-madd -D_REENTRANT -fsigned-char";;
 	*-*-linux*)
 		DEBUG="-g -Wall -W -D_REENTRANT -D__NO_MATH_INLINES -fsigned-char"
-		CFLAGS="-O20 -ffast-math -D_REENTRANT -fsigned-char"
-		PROFILE="-pg -g -O20 -ffast-math -D_REENTRANT -fsigned-char";;
+		CFLAGS="$CFLAGS -ffast-math -D_REENTRANT -fsigned-char"
+		PROFILE="-pg -g $CFLAGS -ffast-math -D_REENTRANT -fsigned-char";;
 	sparc-sun-*)
 		DEBUG="-g -Wall -W -D__NO_MATH_INLINES -fsigned-char -mv8"
 		CFLAGS="-O20 -ffast-math -D__NO_MATH_INLINES -fsigned-char -mv8"
