diff -urN libvorbis-1.0rc1.org/configure.in libvorbis-1.0rc1/configure.in
--- libvorbis-1.0rc1.org/configure.in	Wed Jun 27 12:15:22 2001
+++ libvorbis-1.0rc1/configure.in	Wed Jun 27 12:16:11 2001
@@ -76,9 +76,9 @@
 	case $host in 
 	*86-*-linux*)
 		DEBUG="-g -Wall -W -D_REENTRANT -D__NO_MATH_INLINES -fsigned-char"
-		CFLAGS="-O20 -ffast-math -mno-ieee-fp -D_REENTRANT -fsigned-char"
+		CFLAGS="$CFLAGS -ffast-math -mno-ieee-fp -D_REENTRANT -fsigned-char"
 #              	PROFILE="-Wall -W -pg -g -O20 -ffast-math -D_REENTRANT -fsigned-char -fno-inline -static"
-		PROFILE="-Wall -W -pg -g -O20 -ffast-math -mno-ieee-fp -D_REENTRANT -fsigned-char -fno-inline"
+		PROFILE="-Wall -W -pg -g $CFLAGS -ffast-math -mno-ieee-fp -D_REENTRANT -fsigned-char -fno-inline"
 
 		# glibc < 2.1.3 has a serious FP bug in the math inline header
 		# that will cripple Vorbis.  Look to see if the magic FP stack
@@ -121,8 +121,8 @@
   		fi;;
 	*-*-linux*)
 		DEBUG="-g -Wall -W -D_REENTRANT -D__NO_MATH_INLINES -fsigned-char"
-		CFLAGS="-O20 -ffast-math -D_REENTRANT -fsigned-char"
-		PROFILE="-pg -g -O20 -ffast-math -D_REENTRANT -fsigned-char";;
+		CFLAGS="$CFLAGS -ffast-math -D_REENTRANT -fsigned-char"
+		PROFILE="-pg -g $CFLAGS -ffast-math -D_REENTRANT -fsigned-char";;
 	sparc-sun-*)
 		DEBUG="-g -Wall -W -D__NO_MATH_INLINES -fsigned-char -mv8"
 		CFLAGS="-O20 -ffast-math -D__NO_MATH_INLINES -fsigned-char -mv8"
