--- libvorbis-1.0/vorbis.m4.orig	2002-07-09 15:08:57.000000000 +0200
+++ libvorbis-1.0/vorbis.m4	2003-10-17 20:51:40.371401720 +0200
@@ -54,6 +54,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <vorbis/codec.h>
+#include <vorbis/vorbisenc.h>
 
 int main ()
 {
@@ -62,7 +63,7 @@
     vorbis_info		vi;
 
     vorbis_info_init (&vi);
-    vorbis_encode_init (&vi, 2, 44100, -1, 128, -1);
+    vorbis_encode_init (&vi, 2, 44100, -1, 128000, -1);
     vorbis_analysis_init (&vd, &vi);
     vorbis_block_init (&vd, &vb);
     /* this function was added in 1.0rc3, so this is what we're testing for */
